src/constructor.jl

```diff
@@ -56,6 +56,7 @@ function construct_document(constructor::Constructor, node::Node)
     data
 end
 
+construct_document(::Constructor, ::Nothing) = nothing
```

Comment:
Sorry for my late comment.

Reply:
Yes, there's room for improvement. `collect(load_all("null"))` should definitely return `[nothing]` rather than an empty list but for `load` it's fine to conflate a an empty file with one which contains only a null. Please let's not annoy users by giving an error just because we can.